app/models/member.rb

```diff
@@ -114,6 +117,10 @@ def student_id=(student_id)
     write_attribute(:student_id, nil) if student_id.blank?
   end
 
+  def language=(language)
```

Comment:
This should not be necessary if the attribute is present in the database.

Reply:
Fixed